tests/test_bootstrap.py

```diff
@@ -825,3 +828,65 @@ async def test_bootstrap_empty_integrations(
     """Test setting up an empty integrations does not raise."""
     await bootstrap.async_setup_multi_components(hass, set(), {})
     await hass.async_block_till_done()
+
+
+@pytest.mark.parametrize(
+    ("integration", "dependencies"),
+    [
+        ("mqtt_eventstream", {"file_upload", "http"}),
+        ("mqtt_statestream", {"file_upload", "http"}),
```

Comment:
We should patch async_get_integrations to return mocked manifests in this test so the test won't break if we every change these integrations

Reply:
I might need some help with this. The function needs to return keyed `Integration` classes.

Reply:
https://github.com/home-assistant/core/blob/88bde2a9143b67b4a92ba7d8c8e366bb22c48387/tests/helpers/test_translation.py#L149

You could borrow from this test

Reply:
Thanx for the suggestion, but it seems `homeassistant.helpers.translation.async_get_integrations` is not called during the test.

Reply:
I think `homeassistant.loader.async_get_integrations` should be patched instead

Reply:
> I think `homeassistant.loader.async_get_integrations` should be patched instead

Yes. That test was only intended as an example

Reply:
Mocking async_get_integrations had some impact, but I managed to get it working